lib/beaker/dsl/install_utils/module_utils.rb

```diff
@@ -137,7 +137,7 @@ def copy_module_to(one_or_more_hosts, opts = {})
                 end
               end
             when 'rsync'
-              rsync_to host, source, File.join(target_module_dir, module_name), {:ignore => ignore_list}
+              rsync_to host, source_path, File.join(target_module_dir, module_name), {:ignore => ignore_list}
```

Comment:
Looks like you've got a ride-along here for BKR-189, could you indicate that in your commit message, please?